src/contributors/joaolfelicio.json

```
@@ -0,0 +1,6 @@
+{
+    "name": "Joao Felicio",
+    "desc": "Software Engineer in love with programming.",
+    "github" : "https://github.com/joaolfelicio",
+    "linkedin" : "https://www.linkedin.com/in/joaolfelicio/"
```

```suggestion
    "linkedin": "https://www.linkedin.com/in/joaolfelicio/"
```
Remove unwanted `white-space`.